src/test_utils/katana/genesis.rs

```diff
@@ -203,6 +203,8 @@ impl KatanaGenesisBuilder<Loaded> {
         let proxy_class_hash = self.proxy_class_hash()?;
         let precompiles_class_hash = self.precompiles_class_hash()?;
 
+        let block_gas_limit = FieldElement::from(20_000_000u64);
```

Comment:
Why this value?

Reply:
same value as kakarot

Reply:
when you say Kakarot you mean what we deployed?

Reply:
I'm just not sure where this value comes from